assets/app/view/game/game_info.rb

```diff
@@ -62,6 +62,20 @@ def phases
           phase[:events]&.each do |name, _value|
             event_text << (@game.class::EVENTS_TEXT[name] ? "#{@game.class::EVENTS_TEXT[name][0]}*" : name)
           end
+          trains = @depot.upcoming.select { |train| train.name == phase[:on] }
+          trains.each.with_index do |train, index0|
+            train.events.each do |name|
+              index = index0 + 1
+              ordinal =
```

Comment:
prefer not to do this, if needed make a library or change the verbiage of the text to not depend on this

event_text when purchasing train.name train #index